train.py

```diff
@@ -95,9 +95,6 @@ def train(args):
                 start = time.time()
                 x, y = data_loader.next_batch()
                 feed = {model.input_data: x, model.targets: y}
-                for i, (c, h) in enumerate(model.initial_state):
```

Comment:
Hi, I take a look at the recent sources of tensorflow( [`rnn_cell_impl.py`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/rnn_cell_impl.py) and `seq2seq.py`(?)) and the code seems changed a lot. In my experiments with r11(?) it's safe to remove these lines to have reasonable results. I'm not 100% sure about the way of passing states to a lstm cell, as we are using `self.initial_state` in `rnn_decoder` in `model.py` which is not(?) updated across the batches. If we want to pass the states from the last batch to the next batch, maybe there are more lines to be changed?